lib/deprecated/zbuff_compress.c

```diff
@@ -15,6 +15,7 @@
 ***************************************/
 #define ZBUFF_STATIC_LINKING_ONLY
 #include "zbuff.h"
+#include "error_private.h"
```

Comment:
note : I believe we have decided to use relative `#include` path in this repository, so that should be `../common/error_private.h`